wiki/osu!tourney/kr.md

```diff
@@ -0,0 +1,353 @@
+**osu!tourney** 은 멀티 플레이 경기를 동시에 관전하고, 스트리밍 하기 위해 제작된 multi-spectator 클라이언트입니다.
+
+이 프로그램를 엑세스 할 수 있는 권한은 토너먼트 스텝만이 접근이 가능합니다. 토너먼트 스텝이 되기 위해서는 **tournaments@ppy.sh** 에 연락하여 승인을 받아야합니다.
+사용기한은 토너먼트(대회)가 끝나는 시점으로 종료 됩니다. 
+
+**토너먼트 프로그램 사용시 스트리밍을 하는 대회, 멀티유저 이벤트 이외의 클라이언트 사용을 금지합니다. 또한 클라이언트는 개인용으로 사용할 수 없습니다.**
+
+사용의 **문제**가 있는 경우  **[이 스레드](https://osu.ppy.sh/forum/t/327120)**에서 **[문제해결](Osu!tourney#Troubleshooting "wikilink")** 섹션을 읽거나, 게시 할 수 있습니다. 
+
+osu!tourney 무엇인가?
+===================
+
+osu!tourney 클라이언트는 멀티방에서 모든 플레어어의 플레이를 동시에 관전하는데 사용되는 공식 스트리밍 클라이언트입니다. 클라이언트는 월드컵용으로 제작되었기 때문에 약간의 제약이 있을 수도 있습니다.
+
+토너먼트 클라이언트는 양팀을 합쳐서 16명 까지 지원 가능하며 각 팀 인원은 최소 1명에서 최대 8명 까지 팀을 구성할 수 있습니다. The number of windows opened cannot currently be changed, but this behaviour will be modified in the future.
```

Comment:
Not translated: `The number of windows opened cannot currently be changed, but this behaviour will be modified in the future.`